apps/coursition/prisma/index.ts

```diff
@@ -0,0 +1,2 @@
+import { PrismaClient } from '@prisma/client'
+export default new PrismaClient()
```

Comment:
**blocker:** Default exports should not be used. Also it would be better to have this specified like this:
```typescript
import { PrismaClient } from "@prisma/client"

declare global {
    var prisma: PrismaClient // This must be a `var` and not a `let / const`
}

export const prisma = global.prisma || new PrismaClient()

if (process.env.NODE_ENV !== "production") {
    global.prisma = prisma
}
```

Reply:
Any specific reason for this? 🤔

Reply:
1. Default exports are bad. We should avoid them as much as possible.
2. As for the global client, there's article about it:
https://www.prisma.io/docs/orm/more/help-and-troubleshooting/help-articles/nextjs-prisma-client-dev-practices